allow skip.missing.rhs in calc_addVariable

orichters opened the pull request:
If you try to add a variable where one variable of the right-hand-side does not exist, calc_addVariable fails.
Add switch `skip.fails`, if set to `TRUE`, just skip this calculation.

0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q reviewed (commented):
https://github.com/orichters/quitte/pull/1

orichters commented:
I added the `silent` option to the documentation and renamed the `F` function in `read.quitte` because `make format` was turning it into the `FALSE` function 🤦 .
